bin/create-venv

```diff
@@ -14,7 +14,7 @@ else
 fi
 
 if ${venv_cmd} "${venv_dir}"; then
-    "${venv_dir}/bin/pip" install --upgrade -r "${venv_reqs}"
+    "${venv_dir}/bin/pip" install --upgrade pip setuptools wheel -r "${venv_reqs}"
```

Comment:
So this will work, but with the caveat that it will attempt to upgrade anything in the venv_reqs file to latest version permitted by any constraints specified ;-)

Usually I would do the requirements file as a separate action... In this case it is fine, so you are good.